RSS Reader/RSS Reader/Default/SceneDelegate.m

```diff
@@ -0,0 +1,50 @@
+//
+//  SceneDelegate.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "SceneDelegate.h"
+#import "RSSRTopicsListViewController.h"
+#import "RSSRNetworkService.h"
+#import "RSSRXMLParser.h"
+#import "RSSRTopicsListPresenter.h"
+
+@interface SceneDelegate ()
+
+@end
+
+@implementation SceneDelegate
+
+
+- (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions API_AVAILABLE(ios(13.0)){
+    UIWindow *window = [[UIWindow alloc] initWithWindowScene:(UIWindowScene *)scene];
+    self.window = window;
```

Comment:
use autorelease to reduce the amount of local variables created

Reply:
Done